Description: Use a sensible NAME in the openmpt123.1 man page
Author: James Cowgill <jcowgill@debian.org>
Bug: https://bugs.openmpt.org/view.php?id=881
Applied-Upstream: https://source.openmpt.org/browse/openmpt/trunk/?op=revision&rev=7211
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/man/openmpt123.1
+++ b/man/openmpt123.1
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.46.4.
 .TH OPENMPT123 "1" "September 2016" "openmpt123 v0.2, Copyright (c) 2013-2016 OpenMPT developers <https://lib.openmpt.org/>" "User Commands"
 .SH NAME
-openmpt123 \- manual page for openmpt123 v0.2, Copyright (c) 2013-2016 OpenMPT developers <https://lib.openmpt.org/>
+openmpt123 \- command line module music player based on OpenMPT
 .SH SYNOPSIS
 .B openmpt123
 [\fI\,options\/\fR] [\fI\,--\/\fR] \fI\,file1 \/\fR[\fI\,file2\/\fR] ...
